python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -309,7 +309,27 @@ def realiseModel(self, cache, dataId, bgModel):
         detector = exposure.getDetector()
         bbox = exposure.getMaskedImage().getBBox()
         image = bgModel.toCcdBackground(detector, bbox).getImage()
-        return (detector.getId(), afwMath.binImage(image, BINNING))
+        return self._collect(exposure, image)
+
+    def _collect(self, exposure, image):
```

Comment:
`_collect` is not the clearest of names to me (especially since other methods are called "collect<Something>"), but it is private, so probably fine as is.